frontend/app/services/MemberService.scala

```diff
@@ -482,9 +482,9 @@ class MemberService(identityService: IdentityService,
   private def createAccount(contactId: ContactId, identityId: String, currency: Currency, paymentForm: CommonPaymentForm) =
     paymentForm match {
       case PaymentForm(_, Some(_), _) | MonthlyPaymentForm(Some(_), _, _) =>
-        Account.stripe(contactId, identityId, currency, autopay = true)
+        Account(contactId, identityId, currency, autopay = true, Stripe, deliveryInstructions = None)
```

Comment:
if the deliveryInstructions is defaulted to None in membership common it might stop this strange situation of having to specify the delivery instructions explicitly to None where it's not relevant.